pandas/types/common.py

```diff
@@ -59,6 +62,17 @@ def is_sparse(array):
     return isinstance(array, (ABCSparseArray, ABCSparseSeries))
 
 
+def is_scipy_sparse(array):
+    """ return if we are a scipy.sparse.spmatrix """
```

Comment:
this will keep trying to import scipy each time that it is called if scipy can *not* be imported. You have to set ``_is_scipy_sparse`` to *something* (rather than ``None``) if its ``None`` to start, so that it is *always* set here.

easiest would be to do

``_is_scipy_sparse = lambda x: False`` so this will just work.